Robot.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.VictorSPX;
+import edu.wpi.first.wpilibj.XboxController;
+
+/**
+* The VM is configured to automatically run this class, and to call the functions corresponding to
+* each mode, as described in the TimedRobot documentation. If you change the name of this class or
+* the package after creating this project, you must also update the build.gradle file in the
+* project.
+*/
+public class Robot extends TimedRobot {
+    private VictorSPX mMotor;
+    private XboxController mController;
```

Comment:
You don't need the VictorSPX motor controller for pneumatics. Also, once this is integrated with the actual [Robot Code](https://github.com/frc-6901/2021-Code/blob/master/src/main/java/frc/robot/Robot.java), you will need to use the Xbox controller that has already been declared.